python/cudf/cudf/core/column/column.py

```diff
@@ -120,6 +120,34 @@ def data_array_view(self) -> "cuda.devicearray.DeviceNDArray":
         """
         return cuda.as_cuda_array(self.data).view(self.dtype)
 
+    @property
+    def _data_array_view(self) -> "cuda.devicearray.DeviceNDArray":
+        """
+        Internal implementation for viewing the data as a device array object
+        without triggering a deep-copy.
+        """
+        arr_obj = SimpleNamespace(
```

Comment:
One more place that could use `cuda_array_interface_wrapper`.

Reply:
Done